packages/storage-plus/src/map.rs

```diff
@@ -123,6 +105,34 @@ where
     }
 }
 
+// TODO: move this to a better helpers location
```

Comment:
Do you have something specific in mind? Maybe create an issue along the way?

Reply:
I will do this before merge.
Can't merge with a TODO (gotta make an issue for it first)